mock-bpa-test/test_bpa.py

```diff
@@ -135,6 +129,7 @@ def _wait_for(self, sock: socket.socket) -> bytes:
         if not rrd:
             raise TimeoutError('Did not receive bundle in time')
         data = sock.recv(65535)
+        print(f'WAIT FOR GOT: {binascii.hexlify(data)}')
```

Comment:
Prefer to use LOGGER with a specific severity level to allow filtering out debug stuff.